.github/workflows/formatting-checks.yml

```diff
@@ -17,8 +17,7 @@ jobs:
         with:
           node-version: "20.10.0"
 
-      - name: Install cspell
-        run: yarn add cspell
-
-      - name: Run cspell
-        run: yarn format:cspell
+      - name: Run formatting checks
+        run: |
+          yarn install
+          yarn format
```

Comment:
I think a better idea is to run checker only which is faster. Something like `eslint --check` etc.

This format script actually writes changes to the source code as it is intended to be run developer side before committing.

Reply:
We need to run at least cspell, eslint, probably tsc as well to make sure it compiles. True the code is changed but nothing get committed and you can see the affected files. I'll have a look.

Reply:
@0x4007 How about https://github.com/gentlementlegen/plugin-template/actions/runs/11233078532/job/31226031158

Reply:
I would make a script in package.json like format:ci

run-p cspell:ci eslint:ci prettier:ci

And then fill those sub scripts out accordingly.